dirs/dirs.go

```diff
@@ -179,7 +183,8 @@ func SetRootDir(rootdir string) {
 		DistroLibExecDir = filepath.Join(rootdir, "/usr/lib/snapd")
 	}
 
-	CoreLibExecDir = filepath.Join(rootdir, "/usr/lib/snapd")
+	CoreLibExecDir = "/usr/lib/snapd"
+	CoreSnapMountDir = "/snap"
```

Comment:
The unit test doesn't try to write into those dirs. They are used for example in the content interface where we check if the mount entries generated are ok. @zyga told me that those should never use SnapMountDir (which might be prefixed) but the mount dir without any prefix so /snap on Ubuntu and /var/lib/snapd/snap on Fedora.